Makefile

```diff
@@ -154,7 +154,7 @@ machine: bots
 # checkout Cockpit's test API; this has no API stability guarantee, so check out a stable tag
 # this needs a recent adjustment for firefox 77 and working with network-enabled tests
 test/common:
-	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 229
+	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git e73fd0d54cff27f0e76444a52ae4f07892cfeed1
```

Comment:
we already have usable tag, that is better than sha :)

Reply:
Thanks just realized that and updated haha.